lms/templates/courseware/courseware.html

```diff
@@ -61,7 +61,10 @@
 <%static:css group='style-student-notes'/>
 % endif
 
-<%include file="../discussion/_js_head_dependencies.html" />
```

Comment:
Of course, discussions are rendered natively on mobile, so this may not be necessary. However, it is good to keep the two files in sync, and in fact it will be much clearer if the chromeless version no longer includes discussion files.